db/migrate/20181121135744_add_full_text_index_title.rb

```diff
@@ -0,0 +1,9 @@
+class AddFullTextIndexTitle < ActiveRecord::Migration[5.2]
+  def up
+    execute "create index dimensions_editions_title on dimensions_editions using gin(to_tsvector('english'::regconfig, COALESCE((title)::text, ''::text)))"
```

Comment:
The expression for the index (`COALESCE((title)::text, ''::text)`) here doesn't exactly match what the search function is using. It would be good to double check that the index is still used when searching?

Reply:
@cbaines I haver updated the index. Thank you for spotting this issue.